v1/ao/internal/reporter/reporter_grpc.go

```diff
@@ -921,7 +920,8 @@ func (r *grpcReporter) updateSettings(settings *collector.SettingsResult) {
 	if !r.isReady() && hasDefaultSetting() {
 		r.cond.L.Lock()
 		r.setReady(true)
-		log.Warningf("AppOptics agent (%v) is ready.", r.done)
+		log.Warningf("Got dynamic settings. The AppOptics APM agent (%v, v%v, go%v) is ready.",
```

Comment:
Looks like we have move the version info from the change @ line 339 to here? :)

Could it be a concern as the version info might not print all the time now? For example, if there's a firewall issue then the user will not be able to see version info? (I assume for go the default log level is `warn` ?)

Reply:
Good point! I'll make some change 👍